pkg/assets/rte/assets.go

```diff
@@ -10,6 +10,9 @@ var SELinuxInstallSystemdServiceTemplate []byte
 //go:embed selinuxpolicy.cil
 var SELinuxPolicy []byte
```

Comment:
let's rename this `SELinuxPolicy410` - we're breaking compat already and we will release 0.7.0

Reply:
Yep, makes sense.